test-performance.sh

```
@@ -0,0 +1,44 @@
+# Test date (adjust based on your usage data)
+SINCE_DATE="20250618"
+
+echo
+echo "Testing with --since $SINCE_DATE"
+echo
+
+echo "1. Testing ORIGINAL version (ryoppippi/ccusage)..."
+echo "   Command: npx github:ryoppippi/ccusage daily --since $SINCE_DATE"
+echo "   Timing..."
+time npx --yes github:ryoppippi/ccusage daily --since $SINCE_DATE > /dev/null 2>&1
+ORIGINAL_TIME=$?
```

Shellcheck (0.10.0)

[warning] ORIGINAL_TIME appears unused. Verify use (or export if used externally).

(SC2034)